src/net_processing.cpp

```diff
@@ -4364,6 +4366,12 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
 
     MaybeSendPing(*pto, *peer, current_time);
 
+    if(pto->IsAddrFetchConn() && current_time - std::chrono::seconds(pto->nTimeConnected) > 4 * AVG_ADDRESS_BROADCAST_INTERVAL) {
```

Comment:
nit, don't hesitate to run clang format on the changes
```suggestion
    if (pto->IsAddrFetchConn() && current_time - std::chrono::seconds(pto->nTimeConnected) > 4 * AVG_ADDRESS_BROADCAST_INTERVAL) {
```

Reply:
done, thanks